### PR TITLE
Handle SuiteNotRunning error in rosie go.

### LINKED ISSUE #1874: rosie go: crash on launching cylc gui for non-running suite
rosie go is throwing an error on trying to launch the cylc gui for a non-running suite:

rosie-crash

We should handle this error.

### PR DESCRIPTION
**Pull request by arjclark**
Closes #1874 

Adds a dialog to capture the SuiteNotRunning exception being thrown on trying to launch gcylc from withing rosie go for a non running suite.

@matthewrmshin - please review 1
@kaday - please review 2

### CODE REVIEW

**Comment by kaday** [comment]
Looks good to me.
Passes test battery in my environment.

File: lib/python/rosie/browser/__init__.py [hunk 1]
```diff
@@ -160,6 +160,8 @@
     "\"{0}\". This will permanently delete all branches, " +
     "trunk and any local copy." +
     "\n\nAre you sure you wish to proceed?")
+DIALOG_MESSAGE_SUITE_NOT_RUNNING = ("Cannot launch gcontrol: " +
+                                    "suite {0} does not appear to be running.")
```
Comment (matthewrmshin) [inline lib/python/rosie/browser/__init__.py]: A minor point: the message is basically the same as what is returned by `SuiteNotRunningError.__str__`. I am fine with this change as long as we think this is acceptable.

Comment (arjclark) [inline lib/python/rosie/browser/__init__.py]: Just tweaking slightly to use the __str__ instead to reduce redundancy.

**Comment by arjclark** [comment]
@matthewrmshin - feedback addressed. If you're happy I think we're go for merge.

**Comment by arjclark** [comment]
@matthewrmshin - feedback addressed. If you're happy I think we're go for merge.

**Pull request merged**